README.md

```diff
@@ -85,6 +85,12 @@ Integration tests can be run on any deployed DC/OS cluster. For installation ins
 
 Integration tests are installed via the [dcos-integration-test](./packages/dcos-integration-test/) Pkgpanda package.
 
+Integration test files are stored on the DC/OS master node at `/opt/mesosphere/active/dcos-integration-test`.
+Therefore, in order to test changes to test files, move files from `packages/dcos-integration-test/extra/` in your checkout to `/opt/mesosphere/active/dcos-integration-test` on the master node.
+
+The canonical source of the test suite's results is the continuous integration system.
```

Comment:
This note is the result of a discussion with @jgehrcke .

I wrote this documentation as I used the new instructions to run the IAM tests for the enterprise
After writing this documentation, I found that some tests failed.
For example `test_enterprise_if_marathon_pods_can_be_deployed_with_mesos_containerizer`.

I discussed the following options with @jgehrcke:

* Find and fix the problems with the tests now
* Submit as is
* Don't add documentation now as the tests aren't working, and document this at a later date

The conclusion was that:
* This work was not scheduled, and I did it as a nice-to-have, so the first option (which may take quite some time) is not best
* The third option leaves new developers like me in a worse position than with these docs. For context, it seems that most of the security team is using a similar but custom system maintained by each developer.

I am open to the idea of putting this in the DC/OS Enterprise private repository until this is a fully-working solution, and in fact I have a corresponding PR there.